### PR TITLE
Express Route IPv6 changes #924393

### PR DESCRIPTION
**Pull request by vinayada1**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by markcowl** [comment]
@vinayada1 You are gettign breaking change violations in this PR.  Have you had a cmdlet review?  Please send the changed cmdlet syntax to azdevxps for a cmdlet review before submitting a PR.

File: src/ResourceManager/Network/Commands.Network/packages.config [hunk 1]
```diff
@@ -22,4 +22,4 @@
   <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.7" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
   <package id="System.Spatial" version="5.6.4" targetFramework="net45" />
-</packages>
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/packages.config]: remove this change

File: src/ResourceManager/Compute/Commands.Compute/packages.config [hunk 2]
```diff
@@ -27,4 +27,4 @@
   <package id="System.Net.Requests" version="4.0.11" targetFramework="net45" />
   <package id="System.Spatial" version="5.8.2" targetFramework="net45" />
   <package id="WindowsAzure.Storage" version="8.1.1" targetFramework="net45" />
-</packages>
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute/packages.config]: remove this change

Comment (vinayada1) [inline src/ResourceManager/Compute/Commands.Compute/packages.config]: done

File: src/ResourceManager/Network/Commands.Network/packages.config (hunk 1 above)
Comment (vinayada1) [inline src/ResourceManager/Network/Commands.Network/packages.config]: done

**Comment by markcowl** [comment]
@vinayada1 The issue with the breakign change tests was that your added lines were not in csv format.  Added a commit to fic this.  One change requested (from previous review)

**Pull request merged**